docs/sidebars.js

```diff
@@ -196,6 +191,12 @@ module.exports = {
         'jupyter-notebooks',
       ],
     },
+    {
+      type: 'category',
+      label: 'Reference',
+      collapsed: true,
+      items: [require('./docs/reference/sidebar.json')],
+    },
```

Comment:
Keeping this collapsible since the telemetry page will also go here. Glossary could but i want it easily accessible/noticable to users getting started with Building Assistants, so I left it where it is rn -- lmk your thoughts

Reply:
Seems fine - let's double check how it all looks once the sidebar styling is done